js/components/latestPost/latestPostList.js

```diff
@@ -3,14 +3,14 @@ function renderLatestpost(selector, data) {
 
   let html = '';
   for (const item of data) {
-    html += `<div class="col-12 col-md-4">
-    <div>
+    html += `<div class="col-12 col-md-4 blog">
+    <div class='links'>
     <img src="../../../img/blog-img${item.img}.webp" alt="${item.altText}" class="sizing">
-    <a href="#"><img src="../../../img/blog-${item.thumb}-img.webp">${item.link}</a>
+    <a href="#"><img src="../../../img/blog-${item.thumb}-img.webp"></a><a class='link'>${item.link}</a>
```

Comment:
kodel vardui ir nuotraukai sukurei atskirus linkus? juk paspaudus ant autoriaus turetu mest i jo irasa, nesvarbu ar mini nuotraukyte spaudi ar varda, cia turi but viena nuoroda. plius sitai daliai truksta vienijancio tevinio elemeto, nes net ant didelio ekrano isijungus inspekta jie nebetelpa ir issibegioja
<img width="1440" alt="Screen Shot 2022-04-23 at 12 35 14 PM" src="https://user-images.githubusercontent.com/80914644/164889139-c88ac34e-3255-4790-bbc4-fa149926bb88.png">

Reply:
zinau, kad ant vieno linko turejo buti kaip ir pries tai buvau padarius, bet kadangi neisejo atstumti vardo nuo foto tai priejau toki sprendima, gal zinai kaip atsistumti ant vieno linko? o del to isibegiojimo, tai ir pastebek, kad originale nuo 992px taip pat kuri laika sios dalys buna dvejose eilutese, tiktai ten labiau susiglaudzia ir mazesniam ekrane floatinasi i desine, tai sita reiks pataisyt

Reply:
atstumti paprasta, idek space arba du
kai atidarai a taga <a> iskart uz varneles dedi tarpa(2, 3 kiek nori) ir tada ${item.link}  rasai

Reply:
zodziu, tarp > ir $ simboliu idek tiek space kiek tarpo nori

Reply:
taip, tu teisi, ant media tikrai issibegioja, nemaciau pries tai, tai galima tada toj vietoj ne su flexu juos isstumdyt, o su float suzaiti, bet bet kuriuo atveju ant large ekrano taip neturi but :)

Reply:
> taip, tu teisi, ant media tikrai issibegioja, nemaciau pries tai, tai galima tada toj vietoj ne su flexu juos isstumdyt, o su float suzaisti, arba imti mano siulyta varianta ir tiesiog flex-wrap:wrap uzdet, bet bet kuriuo atveju ant large ekrano taip neturi but :)

nenene su float vel buvo negerai, nes ant mazesnio ekrano floatina i desine, o ant didesnio jau reik jiems i kaire varyt :)

Reply:
> zodziu, tarp > ir $ simboliu idek tiek space kiek tarpo nori

nemaciau tavo patarimo kai lauziau galva, tai pasirodo tiesiog paselektinus img jis puikiausiai atstumia nuo saves net salia jo esancius elementus, galvojau is pradziu, kad inline'ai taip neveikia :)

Reply:
> taip, tu teisi, ant media tikrai issibegioja, nemaciau pries tai, tai galima tada toj vietoj ne su flexu juos isstumdyt, o su float suzaisti, arba imti mano siulyta varianta ir tiesiog flex-wrap:wrap uzdet, bet bet kuriuo atveju ant large ekrano taip neturi but :)

tai ant didelio ekrano viskas vienoj linijoj, o ant vidutinio atsiskiria, bet manau kitaip ir negali buti, nes tekstas kitaip netelpa juk, arba atskiri linkai persikelia vienas po kitu, arba jeigu dviem stulpeliais, tada vistiek dvi eilutes uzimtu

Reply:
sitoj vietoj padding panaikink ir bus perfect
<img width="1440" alt="Screen Shot 2022-04-23 at 3 47 12 PM" src="https://user-images.githubusercontent.com/80914644/164895248-48c44111-fe11-4614-a869-38dd8f337112.png">

Reply:
O va blyn praleidau :))